src/constants.ts

```diff
@@ -81,6 +81,16 @@ export const DEFAULT_DATASETS = [
   url: 'data/weball26.json',
   id: 'weball26',
   group: 'sample'
+}, {
+  name: 'US-state-capitals',
```

Comment:
Can you show a screenshot of what these look like in voyager

Reply:
US:
![image](https://user-images.githubusercontent.com/19829334/123353317-3f78e980-d52f-11eb-850d-17ddbc3e9c41.png)

![image](https://user-images.githubusercontent.com/19829334/123353339-4c95d880-d52f-11eb-839b-8c57e5fb0721.png)

OHLC:
![image](https://user-images.githubusercontent.com/19829334/123353392-6afbd400-d52f-11eb-8899-cc7033ababe7.png)


Main:
![image](https://user-images.githubusercontent.com/19829334/123353415-78b15980-d52f-11eb-98da-30957e9215de.png)

Reply:
They are not useful I think. Please remove them.

Reply:
By them, do you mean just US or both US and OHLC data?

Reply:
(poke in case you missed it)